CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
-
+### tests update
```

Comment:
Can you change this L3 header to one of these:

https://github.com/newrelic/k8s-metadata-injection/blob/main/CONTRIBUTING.md#pull-requests

I think "enhancement" is a good option for this kind of pr.